story.html

```diff
@@ -0,0 +1,155 @@
+                        <p>Many of our riders need a driver after losing full protection of the Voting Rights Act in 2013, as state governments no longer need permission to change election rules. The 2016 US Presidential election will be the first since this change has happened.</p>
+                        <p>Under the new rules, voters in several states face new barriers to getting to the polls. Some of the obstacles that Carpool Vote aims to reduce include:</p>
+                        <ul>
+                            <li>Polling stations face closure in lower-income communities of color,</li>
+                            <li>Would-be voters have to travel out of town to get a photo ID to register,</li>
+                            <li>The periods available to register and vote become shorter, and</li>
+                            <li>Requesting an absentee ballot or voting by mail can be difficult in many states.</li>
+                        </ul>
+                        <p>These rules - along with others introduced since the change - affect everybody but especially people of color, people with disabilities, young people, elderly people, and women.</p>
+                        <button class="button button--large">
+                            <a href='https://medium.com/@RightToVote'>
+                                FIND OUT MORE ABOUT VOTER SUPPRESSION
+                            </a>
+                        </button>
+                        <p>These rules in swing states could sway 84 Electoral College votes and control of the Senate. It looks like the upcoming Presidential Election will be a close fight, so our mission is to help make sure that no vote goes uncounted.</p>
+                        <p>While we’re at it, we love that we can help people of all ages get excited about the political process, and introduce people from all walks of life who may not have had the chance to meet otherwise.</p>
+                        <p>We hope that you love this, too!</p>
+                        <button class="button button--large">
+                            <a href='https://medium.com/@CarpoolVote'>
+                                CARPOOL VOTE NEWS STORIES
+                            </a>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <footer class="footer">
+        <div class="wrapper">
+            <p class="footer__text">
+                Contact us: <a class="footer__link" href="mailto:together@carpoolvote.com">together@carpoolvote.com</a>
+            </p>
+            <form class="newsletter">
+                <label class="newsletter__label" for="signup">Sign up for our newsletter</label>
+                <input class="newsletter__input" type="text" placeholder="email address">
+                <input class="newsletter__submit" value="Join" type="submit">
+            </form>
+        </div>
+    </footer>
+
+</body>
```

Comment:
Looks like the footer analytics tracking code may have been lost with the rest of the JS. Are you able to restore it? Sorry! If it's too much effort let me know and I'll take care of it later on.